<!-- 這個是跑baseline用的 -->
<launch>

    <arg name="save_path" default="$(find localization)/results/base_results_map2car.csv" />
    <param name="use_sim_time" value="true" />

    <node pkg="rviz" type="rviz" name="display_result" output="screen" args="-d $(find localization)/config/itri.rviz" />

    <node pkg="localization" type="pub_map" name="pub_map" output="screen" >
		<param name="map_path" value="/home/louis/sdc_ws/data/itri_map.pcd" />
	</node>

    <node pkg="localization" type="base_localizer" name="base_localizer" output="screen">
        <rosparam file="$(find localization)/config/itri.yaml" command="load" />
        <rosparam param="result_save_path" subst_value="True">$(arg save_path)</rosparam>
    </node>
    

</launch>
